tensorflow_recommenders_addons/dynamic_embedding/python/ops/tf_patch.py

```diff
@@ -56,7 +56,10 @@ def update_op(self, optimizer, g):
         self._v.params.restrict_policy._track_optimizer_slots(_slots)
 
       with ops.control_dependencies([g]):
-        _before = [self._v.read_value()] + [_s.read_value() for _s in _slots]
+        v0 = self._v.read_value(do_prefetch=not self._v.params.bp_v2)
+        s0 = [_s.read_value() for _s in _slots]
```

Comment:
Why not set `do_prefetch` for slots?

Reply:
slots are first initialized here, while the *primary* variables are already initialized in fp.